Add person-attributes-recognition-crossroad-0234 and person-attributes-recognition-crossroad-0238 (#1364)
openvinotoolkit/open_model_zoo: models/intel/person-attributes-recognition-crossroad-0238/description/person-attributes-recognition-crossroad-0238.md

# person-attributes-recognition-crossroad-0238

## Use Case and High-Level Description

This model presents a person attributes classification algorithm analysis scenario.

> The model consists of the Inception V3 backbone and a head, which returns for an input image with a pedestrian 7 values

Reviewer: The sentence looks too complex to me. And the head doesn't work with a bare image.
```suggestion
The model has the Inception V3 backbone. For an image of a person the model returns 7 values
```
The same is for another net.

Author: Ok, I will fix this in a similar way.

Author: Fixed in a similar way.
